tests/RelationsTest.php

```diff
@@ -35,6 +35,7 @@ public function tearDown(): void
         Photo::truncate();
         Label::truncate();
         Skill::truncate();
+        Soft::truncate();
```

Comment:
This was a latent bug when the test suite was run in a particular order, `MassPrunableTest::testPruneSoftDelete` failed.